README.md

```diff
@@ -2,7 +2,7 @@
 
 # OpenTelemetry Operator for Kubernetes
 
-The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/).
+The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](http://web.archive.org/web/20201201093158/https://coreos.com/operators/).
```

Comment:
I would prefer to use this link https://kubernetes.io/docs/concepts/extend-kubernetes/operator/